test/unit/DCAStrategies/dca-strategies-positions-handler.spec.ts

```diff
@@ -65,10 +115,15 @@ contract('DCAStrategiesPositionsHandler', () => {
       given(async () => {
         await DCAStrategiesPositionsHandlerMock.setTokenShares(SHARES);
 
+        hub['deposit(address,address,uint256,uint32,uint32,address,(address,uint8[])[])'].reset();
+        tokenA.transferFrom.reset();
```

Comment:
We usually move all resets after the snapshot's revert, so that we don't have to call it in every test. Like [here](https://github.com/Mean-Finance/swappers/blob/main/test/unit/swap-adapter.spec.ts#L39)